airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import { faGripLinesVertical } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { ReflexContainer, ReflexElement, ReflexSplitter } from "react-reflex";
+import styled from "styled-components";
+
+import { useDocumentationPanelContext } from "./ConnectorDocumentationContext";
+
+const PanelGrabber = styled.div`
+  height: 100vh;
+  padding: 6px;
+  display: flex;
+`;
+
+const GrabberHandle = styled(FontAwesomeIcon)`
+  margin: auto;
+  height: 25px;
+  color: ${({ theme }) => theme.greyColor20};
+`;
+
+//NOTE: ReflexElement will not load its contents if wrapped in an empty jsx tag along with ReflexSplitter.  They must be evaluated/rendered separately.
+
+export const ConnectorDocumentationLayout = ({ children }: { children: [React.ReactNode, React.ReactNode] }) => {
+  const [left, right] = children;
```

Comment:
Not too sure about how I feel about manipulating the children prop this way. It's certainly the cleanest way, otherwise the children is just the content of the page and the documentation panel content is a prop, for example:

```tsx
<ConnectorDocumentationLayout
  documentationPanelElement={<DocumentationPanel />}
>
  <PageTitle />
  <FormPageContent />
</ConnectorDocumentationLayout>
/>
```

Alternatively, looking at the code it doesn't seem that there are alternatives to the `<DocumentationPanel />` (unless I missed it, so you could just have the children be the page content and provide the DocumentationPanel component directly.

Reply:
Happy to do it that way instead to keep things clean.